gci/models.py

```diff
@@ -1 +1,87 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Students(models.Model):
+
+    identifier = models.IntegerField(default=0, primary_key=True)
+    name = models.CharField(max_length=100)
+    program_year = models.ForeignKey('ProgramYear', on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.student_name
+
+
+class Organizations(models.Model):
+
+    identifier = models.IntegerField(default=0, primary_key=True)
+    name = models.CharField(max_length=100)
+    logo = models.ImageField(upload_to='', blank=True)
+    url = models.URLField(blank=True)
+    summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Students)
+
+    def __str__(self):
+        return self.org_name
```

Comment:
I'm confused here, you have used `name` as field above but here using `org_name` 😕